server/service/src/requisition_line/response_requisition_line/update.rs

```diff
@@ -126,8 +130,17 @@ fn generate(
         option_id: updated_option_id,
     }: UpdateResponseRequisitionLine,
 ) -> (Option<RequisitionRow>, RequisitionLineRow) {
+    let suggested_quantity = generate_suggested_quantity(GenerateSuggestedQuantity {
```

Comment:
Calculating suggested for manual requisition lines... maybe should be invalidating frontend if available_SOH changes... we don't have enough params in frontend to calculate it there... 👀